Formula/dnscontrol.rb

```diff
@@ -0,0 +1,21 @@
+class Dnscontrol < Formula
+  desc "It is system for maintaining DNS zones"
+  homepage "https://stackexchange.github.io/dnscontrol"
+  url "https://github.com/StackExchange/dnscontrol/archive/v0.2.7.tar.gz"
+  sha256 "2889c6bec5c4b77a21bda82a1a3760ad64667f8a30ed009b4750f605c9827598"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    mkdir_p buildpath/"src/github.com/StackExchange"
+    ln_s buildpath, buildpath/"src/github.com/StackExchange/dnscontrol"
+
+    system "go", "build", "-o", "dnscontrol"
+    bin.install "dnscontrol"
+  end
+
+  test do
+    assert_match "dnscontrol 0.2.7-dev", shell_output("#{bin}/dnscontrol version")
```

Comment:
Can we test more functionality of the app? From the [docs](https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula):

> We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. However, a bad test is better than no test at all.

Reply:
It's a bad test case, I knew, I tried a bit, then this was working so thought will fix it later. I  need to create a temporary file, how can i do that, it would he helpful if you can point to some references.